UndertaleModLib/UndertaleBaseTypes.cs

```diff
@@ -10,13 +10,13 @@ namespace UndertaleModLib
     public interface UndertaleObject
     {
         /// <summary>
-        /// Serializes the data file into a specified <see cref="UndertaleWriter"/>.
+        /// Serializes the object into a specified <see cref="UndertaleWriter"/>.
```

Comment:
Use cref here please.

Reply:
Should it be "**the** `UndertaleObject`" or "`UndertaleObject`" ?

Reply:
i'd actually prefer `this`.